packages/salesforcedx-vscode-core/package.json

```diff
@@ -69,6 +69,12 @@
           "when": "view == sfdx.force.tasks.view"
         }
       ],
+      "explorer/context": [
+        {
+          "command": "sfdx.force.create.apex",
+          "when": "explorerResourceIsFolder"
```

Comment:
Does this apply to when clause contexts that are already built in? This is given by vscode